gmvault/Dockerfile

```diff
@@ -1,4 +1,6 @@
-FROM python:2-slim
+FROM alpine:3.10
+
+RUN apk add --no-cache bash py2-pip
```

Comment:
Can you explain the logic here a bit?  Why `apk add py2-pip` instead of just using `FROM python:2-alpine3.10`?

Reply:
This way the image will be smaller. I think this was also the cause in https://github.com/tianon/dockerfiles/blob/master/speedtest/Dockerfile?

Reply:
Ahhhh, yep.  Still haven't dug into why that one went from ~100MB down to ~50MB by doing so (since there's probably something to be learned for the `python` image itself by digging into that). :sweat_smile: :+1:

Reply:
That's because of the static `libpython3.8.a`. On Alpine Linux this file lives only in the [python3-dev](https://pkgs.alpinelinux.org/package/edge/main/x86_64/python3-dev) package.

Reply:
Doh, that's a great catch -- IMO that shouldn't be there in any of those `python` images. :sweat_smile: